src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -489,3 +506,6 @@ def _sql_alter_from(self, table: Table, target_table_key: str, ws_id: int):
             f"('upgraded_from' = '{source}'"
             f" , '{table.UPGRADED_FROM_WS_PARAM}' = '{ws_id}');"
         )
+    def _is_migrated(self, schema: str, table: str) -> bool:
+        index = self._migration_status_refresher.index()
```

Comment:
We need to refresh the migration status at the beginning of the `get_remaining_tables`, right?

Reply:
I moved `get_remaining_tables` into the `refresh_migration_status` task. As part of the refresh task, the migration status gets updated, and then the list of remaining tables is published to the migration dashboard.

Reply:
Maybe change the name to `update_migration_status`? What do you think?

Reply:
I would actually prefer if you move the `self._migration_status_refresher.index()` into the `get_remaining_tables`. And remove it from the `update_migration_status` task

Reply:
Done

Reply:
update teh unit test ;)

Reply:
Updated.